src/server/mock-data/json/vergunningen.json

```diff
@@ -1135,7 +1135,7 @@
       "dateDecision": "2023-07-28T00:00:00",
       "licensePlates": "BB-344-P, VV-899-C",
       "previousLicensePlates": "YY-123-A, ZZ-122-A",
-      "requestType": "Kenteken wijziging",
+      "requestType": "wijziging",
```

Comment:
Er wordt overal met `W`ijziging gewerkt ipv `w`ijziging

Reply:
Dan moeten we dat eerst aanpassen in onze decos API.

Reply:
Het lijkt erop dat de snaps daarom ook zijn aangepast https://github.com/Amsterdam/mijn-amsterdam-frontend/pull/976/files#diff-26e39332d805d635678a3ab2ba8dcd060229e7adb31c35ff7ffd8c6bea1e088bL10-L18 https://github.com/Amsterdam/mijn-amsterdam-frontend/pull/976/files#diff-4da10de4a00cb28b93bcaf1b778f788931479195464c2d1342d028ec2cfa4bd6R257-R280 of zie ik dat verkeerd?

Reply:
Ah shit dit is de mock-data. Heb de casing nu aangepast naar hoe het uit de API komt.